diffprivlib/tools/utils.py

```diff
@@ -454,11 +459,26 @@ def _var(array, epsilon=1.0, bounds=None, axis=None, dtype=None, keepdims=False,
                                    upper=((upper - lower) ** 2) / 4, random_state=random_state)
     output = dp_mech.randomise(actual_var)
 
```

Comment:
We should discard mechanisms that are written in diffpriv lib.
Handle this Nan case from the output obtained via opendp or before doing a opendp mechanism.